helm/cluster/tests/components/files/containerd_expected_default.toml

```diff
@@ -26,8 +26,6 @@ enable_selinux = false
 
 [plugins."io.containerd.grpc.v1.cri".registry]
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
-    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
```

Comment:
I changed that due to failing tests for registry. This was not something what I touched directly. https://github.com/giantswarm/cluster/actions/runs/17609577298/job/50028163612?pr=623

Reply:
Nice, thanks for doing this! I already had it on my list, but well.